README.md

```diff
@@ -54,6 +54,18 @@ Below we have the `example.bib` source file used in the sample above.
 }
 ```
 
+### Author names parsing
```

Comment:
That's great to have it documented, but it should be placed in the `RenanBr\BibTexParser\Listener` section, because currently the API doc appears first. However the listener features - like this one - seems to be more useful/important, we can work on it after.